torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2295,7 +2323,29 @@ at::Tensor XLANativeFunctions::pow(const at::Scalar& self,
 at::Tensor XLANativeFunctions::prelu(const at::Tensor& self,
```

Comment:
Why don't we just replace prelu with _prelu_kernel?

Reply:
We can definitely do that. Good point, updated.